### ISSUE TITLE
Remove all uses `text-warning` in any quickstart docs

### ISSUE THREAD

**Issue by kevdevlu** [az-digital/az_quickstart#4334]
Remove usage of `text-warning` in any quickstart docs since it fails WCAG contrast standards. 

This is the only one left that i can find: 
https://quickstart.arizona.edu/site-admin/managing-users/user-expire

**Comment by zsaenz** [comment]
This issue might need a discussion. Besides removing references to text-warning in the referenced areas, do we need to fix or change how callout-warning or text-warning works?


Here's 4 ideas I found:
option 1: Keep it as it is, just remove references in our own docs, QS code, and migrations. It will still exist and folks using it will just have to watch their own sites.
Pro/cons: DB Update text-warning, no bootstrap. Least destructive change, basically just does nothing but removes it from our use (but leaves it there if it’s used elsewhere).

option 2: Force a bg on text-warning if it’s against a white backdrop, like callout-white. The callout background color may be something like bg-gray, so the yellow color matches BG accessibility standards (unknown, there’s no match for it on the color chart).
Pro/cons: Might not require a DB Update, requires bootstrap. Yellow text would be accessible against a darkened backdrop. This will not change any references to text-warning outside of a callout.

option 3: Warning callouts don’t use Warning color, they just use a different color. For example, Callout Bloom is a bright red/pink but has the AZ Blue color. My suggestion would be Mesa, which sorta matches the Warning color.
pro/cons: DB Update text-warning, bootstrap. This only changes callouts, not the text-warning

option 4: Overwrite the color wholly to something accessible like mesa as Joshua suggested, which would be global change to bootstrap. Joshua suggested this is what we did for text-info. This can be Mesa or a new color that passes against a white bg.
pro/cons: Dana was wary about a massive global color change as it could be fairly destructive to existing classes. Would not require a backdrop because the new color should already be accessible.

**Comment by aslaymoore** [comment]
I think a change to Bootstrap which points `.text-warning` classes to another accessible color makes the most sense. Especially since the base theme still makes use of warning/info/danger/etc. classes, generally. The CSS solution would be more of a catch-all and wouldn't require database updates or overriding Barrio to account for their alerts/etc. that they use.
